pages/blog/_posts/2020-10-01-what-is-short-messages.md

```diff
@@ -155,21 +159,32 @@ instead of 140 characters, you can get up to 160 characters per
 *SMS*. The drawback is that you will have a smaller subset of
 characters to use, only the most common is supported. If you include
 any non-*GSM7* characters in your *SMS* then the *SMS* will
-automatically be converted to use *USC-2* instead. *USC-2* uses
-2-bytes, or 16 bits, instead of *GSM7*s 7 bits. That leaves you with
-70 characters per *SMS*. *USC-2* is similar for the basic multilingual
-plane to *UTF-16*. In fact *UTF-16* is an extension of *UCS-2*. The
-main difference is that *USC-2* is fixed width, while *UTF-16* is
-variable width of one or two 16-bits code points. Most phones will
-however see *USC-2* text and think it is *UTF-16* and thus decode it
-wrongly.
+automatically be converted to use either *USC-2* or *GSM7* with a
+different charset instead. *USC-2* uses 2 bytes, or 16 bits, instead
+of *GSM7*s 7 bits. That leaves you with 70 characters per
+*SMS*. *USC-2* is similar for the basic multilingual plane (*BMP*) to
+*UTF-16*. In fact *UTF-16* is an extension of *UCS-2*. The main
+difference is that *USC-2* is fixed width and does not allow for the
+extended characters in the private use area of *BMP*. *UTF-16* is
+variable width of one or two 16-bits code points, and does allow the
+extended characters. For extended characters to work (for instance
+"praying/folded hands" &#x1F64F;), phones might try to fake *UTF-16*
+by using two *USC-2* characters.
+
+Most phones will however see *USC-2* text and
+think it is *UTF-16* and thus decode it wrongly. If *GSM7* have a
```

Comment:
Still not correct.

In practice, UCS-2 is a subset of UTF-16, so UCS-2 -> UTF-16 is OK, the other way is the problem, so the originating phone might *encode* wrongly, and then the fake-UCS2 -> UCS-2 will be wrong (you'd get two question marks instead of the emoji)... But as both iOS and most android phones already does handle it as UTF-16, only older phones will have a trouble with this.